src/UI/Checkbox.elm

```diff
@@ -70,65 +97,90 @@ type alias Options =
 checkbox : String -> (Bool -> msg) -> Bool -> Checkbox msg
 checkbox label message state =
     Checkbox { message = message, label = label, state = state }
-        { labelVisible = True }
+        { labelVisible = True, size = SelectionControl.SizeSM }
 
 
-{-| Show or hide the checkbox's label.
+{-| Hide the checkbox's label.
 
-    TextField.setLabelVisible True someTextField
+    Checkbox.withHiddenLabel someCheckbox
 
 -}
-withLabelVisible : Bool -> Checkbox msg -> Checkbox msg
-withLabelVisible bool (Checkbox prop opt) =
-    Checkbox prop { opt | labelVisible = bool }
+withHiddenLabel : Checkbox msg -> Checkbox msg
+withHiddenLabel (Checkbox prop opt) =
```

Comment:
I noticed that the sole purpose of this function was to hide the label when we only need the checkbox input appearing so I changed it to constrain it to that specific case. If you know a place where we need to toggle the label to be visible for some reason or you disagree to the change let me know.